File: 0075-sort-colors/0075-sort-colors.py
class Solution(object):
    def sortColors(self, nums):
        freq = [0 for i in range(0,3)]
        for i in nums:
            freq[i] += 1
        hold = 0
        for i in range(len(freq)):
            for j in range(0,freq[i]):
                nums[hold] = i
                hold += 1
        
        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """
        

File: 0075-sort-colors/0075-sort-colors.js
/**
 * @param {number[]} nums
 * @return {void} Do not return anything, modify nums in-place instead.
 */
var sortColors = function(nums) {
    let freq = Array(3).fill(0);
    for(let i = 0; i < nums.length; i++){
        freq[nums[i]]++;
    }
    let hold = 0;
    for(let i = 0; i < freq.length; i++){
        for(let j = 0; j < freq[i]; j++){
            nums[hold] = i;
            hold++;
        }
    }
};